source/beamline/BioXAS/BioXASBeamline.h

```diff
@@ -418,14 +408,9 @@ protected slots:
 	void addControlAsDetector(const QString &name, const QString &description, AMControl *control, bool hiddenFromUsers = false, bool isVisible = true);
 
 	/// Protected constructor.
-	BioXASBeamline(const QString &controlName);
+	BioXASBeamline(const QString &beamlineName, const QString &controlName);
```

Comment:
`beamlineName` is like a short version of the name, `controlName` is like a longer version, more detailed information.

This is primarily used for the messaging purpose